dags/dug_helpers/dug_utils.py

```diff
@@ -47,10 +52,11 @@ def __init__(self, config=None, to_string=True):
             ontology_helper = OntologyHelper(url=annotation_config["ontology_metadata"])
 
             redis_config = {
-                'host': self.config.get('redisgraph', {}).get('host'),
-                'port': self.config.get('redisgraph', {}).get('port'),
-                'password': self.config.get('redisgraph', {}).get('password')
+                'host': os.getenv('REDIS_HOST'),
```

Comment:
@YaphetKG I think ultimately we should centralize our config in a similar way that [this dug pr](https://github.com/helxplatform/dug/pull/138) does. So we can add additional properties to the Config class and instantiate it once. But that is something we can design in the future

Reply:
Ah i see, i think there is a class that's already setup for this  https://github.com/helxplatform/roger/blob/9f7b6a9a1257b4c85f77ca8360f4c7852f5f4b39/roger/Config.py#L13

Reply:
There is, but it's difficult to use and reason about. You can't look at the definition in that file and determine what the configuration options the airflow dags expect are.

Reply:
I'm also not sure it's behaving as expected when translating environment variables

Reply:
Got it, thanks for clarifying why we should change makes perfect sense. 
I think we might need to change the name of variables in https://github.com/helxplatform/devops/blob/develop/helx/charts/roger/templates/roger-k8-executor-generic-configmap.yaml , to REDIS_HOST. These env vars  are passed to every executor pod via this configuration https://github.com/helxplatform/devops/blob/da591bd6fa187fcc3347391707eba3fa129430f4/helx/charts/roger/values.yaml#L56

Reply:
Right, I saw those in the helm chart and have them updated on a branch in the devops repo

I think the best course of action is to merge what we have and make the Config class the next step. We need some updates to use the new Dug config and I don't want to make that in this PR or it will easily get too big